client/reader/tags/style.scss

```diff
@@ -117,7 +119,6 @@
 			text-decoration: none;
 			font-weight: 500;
 			flex-grow: 1;
```

Comment:
Might I also suggest removing flex-grow to prevent the misaligned "Z" button?

<img width="131" alt="image" src="https://user-images.githubusercontent.com/1699996/230254092-ea02ddd9-06c6-440b-889b-5b9c75230180.png">